### PR TITLE
mvynokur: added verifyHelpCenterSearchFunctionality test;

### PR DESCRIPTION
**Pull request by MyroslavaVynokur**

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import static com.softserveinc.ita.rozetka.utils.WebElementUtil.*;
+import static java.lang.String.format;
+
+public class HelpSearchResultsPage {
+
+    private final String linkTitleXpath = "(//h3[@class='h5 ih'])";
+
+    public boolean isOpened() {
+        var searchListXpath = "//div[@class = 'search-list']";
+        waitTillVisible(searchListXpath, 10);
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java:15]: why is this needed when you have isVisible() right after?

File: src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        return $x(format(linkTitleXpath + "[%d]", linkNumber)).text();
+    }
+
+    public long getResultsAmount() {
+        return getCollectionSize(linkTitleXpath);
+    }
+
+    public HelpCenterHeader getHelpCenterHeader() {
+        return new HelpCenterHeader();
+    }
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java:33]: lombok

File: src/main/java/com/softserveinc/ita/rozetka/HomePage.java [hunk 3]
```diff
@@ -60,4 +61,8 @@ public GoodsExchangePage openGoodsExchangePage() {
         waitTillPreloaderInvisible();
         return new GoodsExchangePage();
     }
+
+    public CompanyInformationSection getCompanyInformationSection() {
+        return new CompanyInformationSection();
+    }
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/HomePage.java:67]: lombok

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+import lombok.Getter;
+
+import static com.codeborne.selenide.Selenide.$x;
+import static com.softserveinc.ita.rozetka.utils.WebElementUtil.*;
+import static java.lang.String.format;
+
+@Getter
+public class HelpSearchResultsPage {
+
+    private final String linkTitleXpath = "(//h3[@class='h5 ih'])";
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java:13]: there should be no getter for this

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java (hunk 4 above)
Comment (MyroslavaVynokur) [reply src/main/java/com/softserveinc/ita/rozetka/HelpSearchResultsPage.java:13]: Getter is for field below helpCenterHeader as from HelpSearchResultsPage we should have access to helpCenterHeader.
